docs/3.tutorials/fts/1-skeleton.md

```diff
@@ -0,0 +1,189 @@
+---
+id: skeleton
+title: Skeleton and Rust Architecture
+sidebar_label: Contract Architecture
+---
+
+> In this article, you'll learn about the basic architecture behind the FT contract that you'll develop while following this _"Zero to Hero"_ series.
+> You'll discover the contract's layout and you'll see how the Rust files are structured in order to build a feature-complete smart contract.
+
+:::info New to Rust?
+If you are new to Rust and want to dive into smart contract development, our [Quick-start guide](/develop/quickstart-guide) is a great place to start.
+:::
+
+---
+
+## Introduction
+
+This tutorial presents the code skeleton for the FT smart contract and its file structure.
+You'll find how all the functions are laid out as well as the missing Rust code that needs to be filled in.
+Once every file and function has been covered, you'll go through the process of building the mock-up contract to confirm that your Rust toolchain works as expected.
+
+## File structure
+
+The repository comes with many different folders. Each folder represents a different milestone of this tutorial starting with the skeleton folder and ending with the finished contract folder. If you step into any of these folders, you'll find that they each follow a regular [Rust](https://www.rust-lang.org/) project. The file structure for these smart contracts have:
+
+- `Cargo.toml` file to define the code dependencies (similar to `package.json`)
```

Comment:
`Cargo.toml file to define the code dependencies (similar to package.json)` maybe add (similar to package.json in javascript/npm projects) - people might be unweary of javascript (lucky them) 
    - reasoning is that im following the tutorial that speaks for files and stuff and i see "similar to" a file that's never been mentioned before/after whats this file? (package.json) where is it ? I don't have it :( "Can the devs do something?"